.travis.yml

```diff
@@ -11,8 +10,7 @@ before_script:
 
 install:
   - pip install --upgrade pip
-  - pip install -e .
-  - pip install slackclient  # Optional dependency but Slack tests are skipped without it
+  - pip install -e .[slack]
```

Comment:
I would leave this the way it was, I think that's a little more clear (and the comment itself is important as well)